src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -3592,17 +3592,32 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
         }
     }
 
-    // check if all articulations were handled
+    // write all remaining articulations as other-articulation
     for (const Articulation* a : na) {
-        if (!ExportMusicXml::canWrite(a)) {
+        if (a->isOrnament()) {
```

Comment:
We should also check `canWrite` here.
```suggestion
        if (a->isOrnament() || !ExportMusicXml::canWrite(a)) {
```

Reply:
See  #27422